projects/plugins/jetpack/extensions/blocks/podcast-player/edit.js

```diff
@@ -194,6 +193,12 @@ const PodcastPlayerEdit = ( {
 			guids: selectedGuid ? [ selectedGuid ] : [],
 		} );
 
+		fetchTrackQuantity( {
```

Comment:
Hmm this will run each time say `selectedGuid` changes, but really we should only need to fetch the `jetpack_podcast_helper_tracks_quantity` value once from that endpoint.

At the least I think it'd make sense moving the fetch to its own `useEffect`, but even then it'd be called say each time we insert a podcast player block on the same post.

I'm wondering if we could store that retrieved filter value as a piece of state somewhere? @jeherve do you have any suggestions?

Reply:
I agree with you all. Overall I was also expecting same review comments and I wanted to push it this way only but now only thing i'm not sure of testing instruction. I will explain it below what i'm trying to convey here!

Since `get_tracks_quantity` is a static function, `add_filter` code will only work inside this function only. Because api endpoint is calling this function directly.
https://github.com/Automattic/jetpack/blob/dd22ab72fca0880ae68977319c26da71d1bc3ffb/projects/plugins/jetpack/_inc/lib/class-jetpack-podcast-helper.php#L33-L43

I was checking the docs about `add_filter` but couldn't find what is the best practice to write `add_filter` I mean whether we should write it in the class `__construct` or any class method and so on.

BTW Now I have updated the test instructions,  I would love to learn more about it.